packages/cli-doctor/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@react-native-community/cli-doctor",
```

Comment:
I guess the name should be more of a `cli-plugin-doctor`, just like we did with Metro.

Reply:
Got it. Will do Thanks :)